lesson_19/page_objects/main_page_pack/main_page.py

```diff
@@ -0,0 +1,68 @@
+import time
+
+from lesson_19.page_objects.main_page_pack.main_page_locators import MainPageLocators
+from lesson_19.page_objects.players_page_pack.players_page import PlayersPage
+from lesson_19.utilities.web_ui.base_page import BasePage
+from selenium.common.exceptions import NoSuchElementException
+from lesson_19.page_objects.add_player_page_pack.add_player_page import AddPlayerPage
+
+
+class MainPage(BasePage):
+    def __init__(self, driver):
+        super().__init__(driver)
+        self.__page_locator= MainPageLocators()
+
+    def click_sign_out(self):
+        self.click(self.__page_locator.sign_out_button)
+        return self
+
+    def find_elements_on_page(self):
+        elements_to_check = [
+            self.__page_locator.logo,
+            self.__page_locator.dev_contact_link,
+            self.__page_locator.last_cr_match_button,
+            self.__page_locator.players_button,
+            self.__page_locator.last_cr_player_button
+
+        ]
+        found_elements = []
+        for element in elements_to_check:
+            self.wait_until_element_located(element)
+            try:
+                x = self.driver.find_element(by=element[0], value=element[1])
```

Comment:
x?
self.wait_until_element_located(element) -> element. You do not need to find it again

Reply:
fixed, thank you!